openlibrary/core/vendors.py

```diff
@@ -102,29 +103,28 @@ def get_products(self, asins, serialize=False, marketplace='www.amazon.com',
         by comma or as a list.
         """
         item_ids = asins if type(asins) is list else [asins]
+        _resources = (self.RESOURCES[resources] if resources
+                      else self.RESOURCES['import'])
+        try:
+            request = GetItemsRequest(partner_tag=self.tag,
+                                      partner_type=PartnerType.ASSOCIATES,
+                                      marketplace=marketplace,
+                                      item_ids=item_ids,
+                                     resources=_resources,
+                                     **kwargs)
+        except ApiException as e:
+            return None
```

Comment:
This needs to, at a minimum, log the error and perhaps increment an error counter in stats. Silently ignoring without logging means that we'll never know how frequently it's occurring.